src/interactive/Dropdown.jsx

```diff
@@ -106,18 +107,34 @@ class Dropdown extends React.PureComponent<Props, State> {
 		}
 	};
 
-	onBodyClick = (e: SyntheticMouseEvent<*>) => {
-		if (this.props.closeOnBodyClick) {
-			this.closeContent(e);
-			return;
+	/** Handles execution of menuItem action and then closes the dropdown
+	 	_if_ `closeOnMenuItemClick` is set to true and there are menuItems.
+	 	If there are menuItems but `closeOnMenuItemClick` is false it'll just
```

Comment:
I think the second sentence isn't needed, it's just describing what the code does